syncer/proto/syncer.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package v1;
```

Comment:
If we want to use message Status, we need to use a different package. Networks proto are in v1 package and already have message Status

Reply:
What do you mean? Do you want to define Status in another proto?

Reply:
All our proto files in polygon-edge repo are inside the same package (**package v1**). In one package you can not have a message with the same name. **identity.proto** already has a message named **Status**. 

We have added in syncer.proto **message Status**. If you try to run it you will get an error that you cannot have the same message name inside one package.

I am suggesting just to move syncer.proto in another package and then we can have same message names in multiple files

Reply:
I understood now. Fixed ef8b18e795c3af1692bc5c5d49c764dc23d64c3a